main.py

```diff
@@ -3,6 +3,10 @@
 from graph_config import get_user
 from external_service import extract_objects
 
+# suppress warnings when working locally with Dev Proxy
+import urllib3
+urllib3.disable_warnings()
```

Comment:
When you disable SSL validation, you get several warning. This suppresses them to keep the output clean.